popupDialogExample/PopupDialogExample.js

```diff
@@ -1,47 +1,65 @@
-import React, { Component } from 'react';
-import { View, Text, StyleSheet } from 'react-native';
-import { RoundButton } from 'react-native-button-component';
+import React, {Component} from 'react';
+import {View, Text, StyleSheet, TouchableHighlight, Dimensions} from 'react-native';
 import PopupDialog from 'react-native-popup-dialog';
 
+const {width, height} = Dimensions.get('window');
```

Comment:
@aviadhahami 
Sorry, It's should belong to ES6.
It called `Destructuring Objects` / `Destructuring Object Property` or something like that.

FYI: https://hacks.mozilla.org/2015/05/es6-in-depth-destructuring/